### PR TITLE
Create user input for number of samples

### PR DESCRIPTION
**Pull request by raplima**
Now users should be able to select the minimum and maximum number of samples per class (part of #39, necessary for #20)

### CODE REVIEW

File: uis/MainWindow.py [hunk 1]
```diff
@@ -86,15 +103,22 @@ def setupUi(self, MainWindow):
...
-        self.pushButton_inputFileLito.setText(QtWidgets.QApplication.translate("MainWindow", "Selecionar Arquivo", None, -1))
+        self.pushButton_Cancel.setText(QtWidgets.QApplication.translate("MainWindow", "Cancel", None, -1))
+        self.label_inputFeatures.setText(QtWidgets.QApplication.translate("MainWindow", "Sensoriamento (features):", None, -1))
+        self.label_inputLimit.setText(QtWidgets.QApplication.translate("MainWindow", "Limite:", None, -1))
+        self.label_output.setText(QtWidgets.QApplication.translate("MainWindow", "Output Folder:", None, -1))
+        self.groupBox_options.setTitle(QtWidgets.QApplication.translate("MainWindow", "Options", None, -1))
+        self.label_maxSamples.setText(QtWidgets.QApplication.translate("MainWindow", "Maximum number of samples per class:", None, -1))
+        self.lineEdit_maxSamples.setToolTip(QtWidgets.QApplication.translate("MainWindow", "Classses with more samples than the number indicated here will be randomly selected.", None, -1))
+        self.lineEdit_maxSamples.setText(QtWidgets.QApplication.translate("MainWindow", "150", None, -1))
```
Comment (marcosbr) [inline uis/MainWindow.py:120]: The maximum value must always be greater than the minimum

Comment (marcosbr) [reply uis/MainWindow.py:120]: Cannot be negative as well

File: uis/MainWindow.py [hunk 2]
```diff
@@ -3,35 +3,30 @@
 # Form implementation generated from reading ui file 'MainWindow.ui',
```
Comment (marcosbr) [inline uis/MainWindow.py:3]: The UI should have a checkbox (previously selected) that the user chooses to run the PCA on the multi-band raster (default value: True) variable name self.run_pca)

File: gui_main.py [hunk 3]
```diff
@@ -23,6 +24,13 @@ def __init__(self, *args, **kwargs):
         self.setupUi(self)
         self.child_win = None
         self.display_wins = {}
+
+        #############################################################################
+        # validations
+        self.onlyInt = QIntValidator()
+        for line_edit in [self.lineEdit_atLeast,
```
Comment (marcosbr) [inline gui_main.py:31]: The value cannot be null or negative

**Review by marcosbr — CHANGES_REQUESTED** [review]

File: uis/MainWindow.py (hunk 1 above)
Comment (raplima) [reply uis/MainWindow.py:120]: Added check for that.

File: gui_main.py (hunk 3 above)
Comment (raplima) [reply gui_main.py:33]: Added check.

File: uis/MainWindow.py (hunk 2 above)
Comment (raplima) [reply uis/MainWindow.py:3]: Out of scope here, will save fo later

**Review by marcosbr — APPROVED** [review]

**Pull request merged**